examples/docs_snippets/docs_snippets_tests/deploying_tests/test_monitoring_daemon_examples.py

```diff
@@ -5,4 +5,4 @@
 def test_run_timeouts():
     assert my_job.execute_in_process().success
     assert my_job.tags[MAX_RUNTIME_SECONDS_TAG] == "10"
-    assert asset_job.tags[MAX_RUNTIME_SECONDS_TAG] == 10
```

Comment:
This change is just from adding tag normalization to `UnresolvedAssetJobDefinition` (where it should've been already anyway).